examples/log-generator/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8.2.1-alpine
+
+WORKDIR /log-generator
+
+ENV ESCRIBA_TIMEOUT 3000
```

Comment:
It's a @drrzmr recommendation to be more explicit and doesn't affect the code at all. Also this is good when you're using docker inspect.